helm-charts/interoperator/values.yaml

```diff
@@ -68,6 +68,7 @@ interoperator:
     provisionerWorkerCount: 2
 
 operator_apis:
+  enabled: false
```

Comment:
not sure if we should keep the default as true or false. Keeping it true would atleast be status quo. Or else, it will suddenly vanish for existing deployments.

Reply:
Ok. Will update to true